ci/do_ci.sh

```diff
@@ -312,19 +312,20 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
   collect_build_profile build
   exit 0
 elif [[ "$CI_TARGET" == "bazel.api" ]]; then
+  # Use libstdc++ because the API booster isn't capable of working with libc++ yet.
+  ENVOY_STDLIB="libstdc++"
```

Comment:
I think this is the part that is confusing me. Can you add more comments here per the other comments? Is this because of RBE or something else? Why doesn't it work and why do we need to set this? Otherwise LGTM.

/wait

Reply:
done